gripper_controller:
  type: position_controllers/JointTrajectoryController
  joints:
     - finger_joint
  constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.05
      simple_gripper_robotiq_85_left_knuckle_joint: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10

  ## joint_state_controller
gripper_joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: &loop_hz 500
# ## joint_group_position_controller
# hand_group_position_controller:
#     type: position_controllers/JointGroupPositionController
#     joints:
#      - $(arg prefix)finger_1_joint_1
#      - $(arg prefix)finger_2_joint_1
#      - $(arg prefix)finger_middle_joint_1
#      - $(arg prefix)palm_finger_1_joi
# ## single command controller
# hand_simple_controller:
#     type: position_controllers/JointPositionController
#     joint: $(arg prefix)finger_joint